### PR TITLE
Added Overview and Getting started UG for PullToRefresh control.

### PR DESCRIPTION
**Pull request by DiwakarVenkatesan**
Added Overview and Getting started UG for PullToRefresh control.

### CODE REVIEW

File: MAUI/Pull-to-Refresh/overview.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: About .NET MAUI PullToRefresh control | Syncfusion
+description: Learn about the introduction of Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control, its elements and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+--- 
+
+# MAUI PullToRefresh (SfPullToRefresh) Overview
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/overview.md:10]: Title should be - Overview of .NET MAUI PullToRefresh (SfPullToRefresh)

File: MAUI/Pull-to-Refresh/overview.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+PullToRefresh is a refresh control that allows you to interact and refresh the view loaded in it. The PullToRefresh control allows you to refresh the view upon performing the pull-to-refresh action. A progress indicator will be shown while starting the pulling. The application will be refreshed once you have pulled down a certain distance and released the touch.
+
+## Use Case Scenarios
+
+`PullToRefresh` controls are mainly used in applications where users need to refresh the content on demand. PullToRefresh controls are mainly used in applications where users need to refresh the content on demand. The main applications that use the PullToRefresh are listed below:
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/overview.md:16]: some sentences are repeated. Check and remove.

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+title: Getting Started with .NET MAUI PullToRefresh control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control, its elements, and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---
+
+# Getting Started with .NET MAUI PullToRefresh (SfPullToRefresh)
+
+This section provides a quick overview of how to get started with the .NET MAUI PullToRefresh (SfPullToRefresh) for MAUI. Walk through the entire process of creating the real-world SfPullToRefresh.
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:12]: and a walk-through to configure the .NET MAUI PullToRefresh control in a real-time scenario.

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+## Creating an application using the .NET MAUI PullToRefresh
+
+ 1. Create a new .NET MAUI application in Visual Studio.
+ 2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfPullToRefresh to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.PullToRefresh and install it.
+ 3. Import the control namespace `Syncfusion.Maui.PullToRefresh` in XAML or C# code.
+ 4. Initialize the [SfPullToRefresh]() control.
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:19]: for now, we can highlight "SfPullToRefresh" word. Remove the format of adding link.

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+ 
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="4" %}
+
+    <ContentPage
+        xmlns:syncfusion="clr-namespace:Syncfusion.Maui.PullToRefresh;assembly=Syncfusion.Maui.PullToRefresh">
+
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:26]: remove unwanted line space or ensure hl_lines="4" is correct index or not

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    using Syncfusion.Maui.PullToRefresh;
+    . . .
+
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfPullToRefresh pullToRefresh = new SfPullToRefresh();
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:41]: missed below line.
this.Content = pullToRefresh

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+                await Task.Delay(2000);
+                this.pullToRefresh.IsRefreshing = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI PullToRefresh with slide on top transition mode](Images/getting-started//maui-pulltorefresh-getting-started.png)
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:131]: pls add .gif images

File: MAUI/Pull-to-Refresh/overview.md (hunk 2 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/overview.md]: Removed the same sentence

File: MAUI/Pull-to-Refresh/getting-started.md (hunk 3 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/getting-started.md]: Added review corrections

File: MAUI/Pull-to-Refresh/getting-started.md (hunk 5 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/getting-started.md]: checked and removed blank space

File: MAUI/Pull-to-Refresh/getting-started.md (hunk 6 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/getting-started.md:40]: Added

File: MAUI/Pull-to-Refresh/getting-started.md (hunk 7 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/getting-started.md]: Changed png to gif

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**